jsonargparse_tests/test_actions.py

```diff
@@ -53,6 +53,10 @@ def test_ActionYesNo_parse_env(self):
         parser.add_argument('--op', action=ActionYesNo, default=False)
         self.assertEqual(True, parser.parse_env({'APP_OP': 'true'}).op)
 
+        parser = ArgumentParser(default_env=True, env_prefix=False)
```

Comment:
Please make the test standalone and don't use an action. The test should be in `test_core.py` since parsing of environment variables has nothing to do with actions.

Also please add a test for the deprecation message in `test_deprecated.py`.

Reply:
Thanks for the pointer - done